### PR TITLE
CM-133 - Moved all addresses from 'cosmos...' notation to 'bluzelle...' notation

### PR DESCRIPTION
**Pull request by njmurarka**

### CODE REVIEW

File: cmd/blzcli/main.go [hunk 1]
```diff
@@ -28,8 +28,12 @@ import (
 	authcmd "github.com/cosmos/cosmos-sdk/x/auth/client/cli"
 	authrest "github.com/cosmos/cosmos-sdk/x/auth/client/rest"
 	bankcmd "github.com/cosmos/cosmos-sdk/x/bank/client/cli"
+
```
Comment (rnistuk) [inline cmd/blzcli/main.go:31]: Please run go format before submitting code

File: types/types.go [hunk 2]
```diff
@@ -0,0 +1,20 @@
+package types
+
+// const (
```
Comment (ebruck) [inline types/types.go]: please remove dead code

Comment (rnistuk) [inline types/types.go]: Remove dead code

File: types/types.go [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+// 	AppName = "CRUD"
+// )
+
+const (
+	// Bech32PrefixAccAddr defines the Bech32 prefix of an account's address
```
Comment (rnistuk) [inline types/types.go:4]: This is a bit worrying, has this been tested in a zone?

Comment (rnistuk) [reply types/types.go:4]: I'll try this out tomorrow.

**Comment by njmurarka** [comment]
I tried it on a one-node zone on my machine with CRUD and sending tokens.
Works fine.

Obviously we do not have integration tests to validate nothing else is
broken.

File: types/types.go (hunk 3 above)
Comment (njmurarka) [reply types/types.go:4]: I tried it on a one-node zone on my machine with CRUD and sending tokens.
Works fine.

Obviously we do not have integration tests to validate nothing else is
broken.

File: types/types.go (hunk 2 above)
Comment (njmurarka) [reply types/types.go]: Done

Comment (njmurarka) [reply types/types.go]: Done

File: cmd/blzcli/main.go (hunk 1 above)
Comment (njmurarka) [reply cmd/blzcli/main.go:31]: I did. There are no differences. 

gofmt -w <file>

**Review by rnistuk — APPROVED** [review]

**Pull request merged**